repos.yaml

```diff
@@ -203,3 +203,8 @@ repositories:
   maintainers:
   - name: Loki Maintainers
     mail: lokiproject@googlegroups.com
+- name: mmlspark
```

Comment:
The name in both files should match. Since this name is used for displays you might consider "Microsoft"

Reply:
@dbanda I was suggesting just "Microsoft". Having charts in the display name is a bit redundant for the helm hub. it's all about charts. Did you want to leave charts on it?

Reply:
@mattfarina thanks for the suggestion. fixed it.